docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md

```diff
@@ -61,6 +61,8 @@ This section describes any UAN details that an administrator must be aware of be
 
 **Action**: Before executing this stage, any site-local UAN configuration changes must be made so that the following stages execute using the wanted UAN configuration values. See the [Basic UAN Configuration](../operations/Basic_UAN_Configuration.md) section of this documentation for UAN configuration content details. The [Prepare for UAN Product Installation](../installation_prereqs/Prepare_for_UAN_Product_Installation.md) section is required for fresh installation scenarios.
 
+The procedures in this guide assume that the HPE Cray Supercomputing EX system has dedicated UANs. If the HPE Cray Supercomputing EX system does not have dedicated UANs, skip the steps for installing and  configuring them. 
```

Comment:
There's an extra space between 'and' and 'configuring' in this sentence.

Reply:
@keopp-hpe I just added a commit to fix it.